src/units/unit.cpp

```diff
@@ -2115,10 +2220,14 @@ void unit::apply_builtin_effect(std::string apply_to, const config& effect)
 		}
 	} else if(apply_to == "remove_ability") {
 		if(const config& ab_effect = effect.child("abilities")) {
+			deprecated_message("remove_ability[abilities][tag name]", DEP_LEVEL::INDEFINITE, "", "Use the remove_ability[filter_ability] instead");
```

Comment:
```suggestion
			deprecated_message("[effect]apply_to=remove_ability [abilities]", DEP_LEVEL::INDEFINITE, "", "Use [filter_ability] instead");
```

Reply:
you don't precise what [filter_ability] must be used in [effect]apply_to=remove_ability

Reply:
change done, but i added " in [effect]apply_to=remove_ability" for what develloper don'timagine what [filter_abilities] used instead of effect]apply_to=remove_ability (someone could unterstand that, but it have few probabilities)